### PR TITLE
Luciendupont

### PR DESCRIPTION
**Pull request by lwdupont**
Added my info.

### CODE REVIEW

**Comment by sojournerjosh** [comment]
Hello Lucien, good bio :-)

File: bios/lucienbio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+* Lucien Dupont
+* Canadian
+* Married.
```
Comment (bmatiu) [inline bios/lucienbio.md:4]: Any Child?

**Comment by lwdupont** [comment]
Yes, good idea :)

File: bios/lucienbio.md (hunk 1 above)
Comment (kennym52) [inline bios/lucienbio.md:4]: You have a song named after you, but you know that of course! :-)

**Pull request merged**